src/components/ContextMenu/PContextMenu.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <PPopOver
+    ref="popOver"
+    class="p-context-menu"
+    :placement="internalPlacement"
+    :group="group"
+    :auto-close="autoClose"
+  >
+    <template #target="{ toggle, open, close, visible }">
+      <slot name="target" v-bind="{ toggle, open, close, visible }" />
+    </template>
+    <template #default="{ close }">
+      <slot name="context-menu" v-bind="{ close }">
+        <POverflowMenu class="p-context-menu__content" @click.stop="closeOnMenuClick">
+          <slot v-bind="{ close }" />
+        </POverflowMenu>
+      </slot>
+    </template>
+  </PPopOver>
+</template>
+
+<script lang="ts" setup>
+  import { useGlobalEventListener } from '@prefecthq/vue-compositions'
+  import { computed, ref } from 'vue'
+  import POverflowMenu from '@/components/OverflowMenu/POverflowMenu.vue'
+  import PPopOver from '@/components/PopOver/PPopOver.vue'
+  import { useMousePosition } from '@/compositions'
+  import { PositionMethod, keys } from '@/types'
+  import { isKeyEvent } from '@/utilities'
+  import { bottomLeft, bottomRight, topLeft, topRight } from '@/utilities/position'
+  import { lowerLeft, lowerRight, upperLeft, upperRight } from '@/utilities/positionRelative'
+
+  const props = withDefaults(defineProps<{
+    placement?: PositionMethod | PositionMethod[],
+    autoClose?: boolean,
```

Comment:
should this be `disableAutoClose` since the other close props are true by default?

Reply:
Mmm yeah you're right, I originally was just re-exposing the `PPopover` prop but I think this makes more sense